check_nightly_success/check-nightly-success/check.py

```diff
@@ -60,14 +64,15 @@ def main(
                 branch_ok[branch] = True
                 break
 
-    failed_branches = [k for k, v in branch_ok.items() if not v]
-    if failed_branches:
-        print(  # noqa: T201
-            f"Branches with no successful runs of {workflow_id} in the last "
-            f"{max_days_without_success} days: "
-            f"{', '.join(failed_branches)}",
-        )
-    return len(failed_branches) > 0
+    # Only check the latest branch (lexicographic ordering is sufficient for max here).
+    latest_branch = max(branch_ok)
+    retcode = not branch_ok[latest_branch]
+    status_msg = "no successful runs" if retcode else "a successful run"
+    print(  # noqa: T201
+        f"{latest_branch} has {status_msg} of {workflow_id} in the last "
+        f"{max_days_without_success} days: "
```

Comment:
@vyasr The trailing colon here makes it seem like there is supposed to be more output. I don't see any more lines in the logs. Was this supposed to print the most recent successful date or something?

Reply:
Addressed in #37.